region/crateria/east/The Moat.json

```
@@ -828,16 +892,86 @@
       "requires": [
         "canSuitlessMaridia",
         "h_canDoubleSpringBallJumpWithHiJump",
-        {"obstaclesNotCleared": ["A"]}
+        {"obstaclesNotCleared": ["A", "B", "C"]}
       ]
     },
     {
       "link": [3, 2],
       "name": "Moat Ceiling Bomb Jump",
       "requires": [
         "canCeilingBombJump",
-        {"obstaclesNotCleared": ["A"]}
+        {"obstaclesNotCleared": ["A", "B", "C"]}
+      ]
+    },
+    {
+      "link": [3, 2],
+      "name": "Continue Shinespark Right",
+      "requires": [
+        {"obstaclesCleared": ["B"]},
+        {"obstaclesNotCleared": ["A", "C"]},
+        {"shinespark": {"frames": 23, "excessFrames": 7}}
+      ],
+      "resetsObstacles": ["B"]
+    },
+    {
+      "link": [3, 2],
+      "name": "Continue Hero Shot Shinespark Right",
+      "requires": [
+        {"obstaclesCleared": ["B", "C"]},
+        {"obstaclesNotCleared": ["A"]},
```

Is this necessary? If not, a horizontal is.
```suggestion
        {"obstaclesNotCleared": ["A"]},
        "canMidairShinespark",
```

---

I think the tech requirements would belong on the other strat that initiated the spark? For this strat the player doesn't have to do anything more, just let the spark continue